wsba-coordinator-completion-simple/README.adoc

```diff
@@ -60,10 +60,12 @@ You must start {productName} with the XTS subsystem enabled.
 +
 [source,subs="+quotes,attributes+",options="nowrap"]
 ----
-$ __{jbossHomeName}__/bin/standalone.sh --server-config=../../docs/examples/configs/standalone-xts.xml | egrep "started|stdout"
+$ __{jbossHomeName}__/bin/standalone.sh --server-config=../../standalone/configuration/standalone-xts.xml | egrep "started|stdout"
 ----
 +
 NOTE: For Windows, use the `__{jbossHomeName}__\bin\standalone.bat` script.
++
+NOTE: If you do not have the standalone-xts.xml in the configuration folder you can copy and paste a working example from the ../../docs/examples/configs/ folder.
```

Comment:
@marcosgopen For my own interest, what was the problem with `--server-config=../../docs/examples/configs/standalone-xts.xml`, does this change mean that it won't be possible to automate testing this quickstart (EAP7-1842 Quickstarts automated functional testing).

Reply:
It is possible to automate the quickstart, but it needs a script that copy the ../../docs/examples/configs/standalone-xts.xml in ../../standalone/configuration/standalone-xts.xml 
In fact when you run standalone with "--server-config=../../docs/examples/configs/standalone-xts.xml" it fails with the following error message:
`java.lang.IllegalStateException: WFLYCTL0214: Could not load configuration file: ../../docs/examples/configs/standalone-xts.xml. The configuration file argument must specify the path to a file located in the configuration directory. The path must be a relative path, and must be relative to the configuration directory ~/wildfly/build/target/wildfly-28.0.0.Beta1-SNAPSHOT/standalone/configuration.` [As you may have seen in the path I tested it with wildfly-28.0.0.Beta1]

Reply:
I assume the sentence "The configuration file argument must specify the path to a file located in the configuration directory" means the xml file should be in wildfly-28.0.0.Beta1-SNAPSHOT/standalone/configuration folder. Might it be a change in the last releases?